.travis.yml

```diff
@@ -0,0 +1,42 @@
+jobs:
+  include:
+    - language: node_js # js testing
+
+      node_js:
+      - "stable" # latest stable node version
+
+      cache: yarn # using yarn to install dependencies
+
+      install:
+        - yarn
```

Comment:
Not sure that this will work. On CI, you need to fetch and build yarn imo!

Reply:
@parthsujalshah check this once